crates/utils/re_uri/src/redap_uri.rs

```diff
@@ -362,7 +362,7 @@ mod tests {
 
     #[test]
     fn test_localhost_url() {
-        let url = "rerun+http://localhost:51234/catalog";
```

Comment:
Although it doesn't change the test, having these recap-centric tests using a non-redap port sends to wrong message. These changes should be reverted imho.